tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/mapper/PluralsMapper.java

```diff
@@ -35,15 +34,21 @@
  * }</pre>
  */
 public final class PluralsMapper {
-    private static final PathMatcher PLURALS = PathMatcher.of("supplementalData/plurals[@type=*]");
-    private static final AttributeKey PLURALS_TYPE = keyOf("plurals", "type");
 
-    private static final PathMatcher RULES = PathMatcher.of("pluralRules[@locales=*]");
+    private static final AttributeKey PLURALS_TYPE = keyOf("plurals", "type");
     private static final AttributeKey RULES_LOCALES = keyOf("pluralRules", "locales");
-
-    private static final PathMatcher RULE = PathMatcher.of("pluralRule[@count=*]");
     private static final AttributeKey RULE_COUNT = keyOf("pluralRule", "count");
 
+    private static final CldrDataProcessor<PluralsMapper> CLDR_PROCESSOR;
+    static {
+        CldrDataProcessor.Builder<PluralsMapper> processor = CldrDataProcessor.builder();
+        processor
```

Comment:
This is really only processing one path (pluralRule) but it's done as 3 nested actions because of the need to group these rules in the generated data. Each addAction provides a new scope in which the child actions will be processed. This makes is clear that:
* Each `Rule` is collected by a `Rules` instance
* `Rules` are collected by a `Plurals` instance
* The mapper (top level) collects `Plurals`